src/lib/cimb_translator/CellDrift.cpp

```diff
@@ -29,3 +29,15 @@ void CellDrift::updateDrift(int dx, int dy)
 	_y = std::min(_y, _limit);
 	_y = std::max(_y, 0-_limit);
 }
+
+// static
+uint8_t CellDrift::calculate_cooldown(uint8_t previous, uint8_t idx)
+{
+	if (idx == 4)
+		return 4;
+	if (idx % 2 == 0)
+		return 0xFF;
+	if ((previous xor idx) == 6) // 1 and 7, 3 and 5
```

Comment:
This can also be written as four (more) `if` statements. I'm not sure which is better. Probably the comment could be improved.